pkg/object/httpserver/spec.go

```diff
@@ -101,6 +102,15 @@ type (
 
 		headerRE *regexp.Regexp
 	}
+
+	// Query is the third level entry
+	Query struct {
+		Key    string   `json:"key" jsonschema:"required"`
+		Values []string `json:"values,omitempty" jsonschema:"omitempty,uniqueItems=true"`
+		Regexp string   `json:"regexp,omitempty" jsonschema:"omitempty,format=regexp"`
+
+		queryRE *regexp.Regexp
```

Comment:
per Golang naming convention, we should not repeat type names in field names.

```suggestion
		re *regexp.Regexp
```